cypress/integration/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script type="text/javascript" src="https://userbase-public.s3-us-west-2.amazonaws.com/userbase-js/userbase.js"></script>
```

Comment:
Would it make sense to load the script locally? Running "npm run build:sdk-script" will build the script in ./build/scripts/userbase.<version>.js

We could make "npm run build:sdk-script" to produce a userbase.latest.js as well so that the filename won't change all the time.

And if we make the tests runnable from a package.json script, we could run build:sdk-script before cypress open to make sure that the latest script is always built.

Reply:
Great idea!

Reply:
Just a head's up, the local version's API is slightly different from the S3 version. The docs here are up to date with the local version

Reply:
Going off this, it would also make sense to run the most recent server too and not use the default server that's provided by the SDK (otherwise you'll run into some issues)

To do this, you would need to run the server before running the test (`npm run watch:server` would work), and then add this to your configure function: `userbase.configure({ appID: <App Id>, endpoint: http://localhost:3000 })`

Reply:
to have a short path I will update the doc, then, will implement the automation.

Reply:
Implemented a `npm run test` command. Added local development setup https://github.com/encrypted-dev/userbase/commit/24d6b8180cdfdab565cee76655229d32dbd578d4